skiko/src/awtMain/kotlin/org/jetbrains/skiko/MetalApi.kt

```diff
@@ -8,6 +8,11 @@ package org.jetbrains.skiko
  */
 internal fun chooseMetalAdapter(adapterPriority: GpuPriority): MetalAdapter {
     val adapter = chooseAdapter(adapterPriority.ordinal)
+
+    if (adapter == 0L) {
+        throw RenderException("MetalApi.chooseAdapter returned null")
```

Comment:
Do we have more appropriate exception type? It's not about rendering but about initialization

Reply:
`RenderException` is used throughout entire skiko for initialisation and is expected in
https://github.com/JetBrains/skiko/blob/38090dc7aef0a8ac83a2cac0f9f8730b40cbf6f7/skiko/src/awtMain/kotlin/org/jetbrains/skiko/redrawer/RedrawerManager.kt#L29-L35
To fallback to software renderer.

Reply:
I think that we need to subclass it for such cases at some point, not a blocker for this PR

Reply:
Probably, out of scope for this fix